src/goto-checker/multi_path_symex_only_checker.cpp

```diff
@@ -74,8 +74,8 @@ void multi_path_symex_only_checkert::generate_equation()
 {
   const auto symex_start = std::chrono::steady_clock::now();
 
```

Comment:
This PR seems like it should be a couple of PRs about different things. This doesn't feel like it belongs in a documentation PR.